src/pages/events/configure-commerce.md

```diff
@@ -0,0 +1,193 @@
+
+Make the following modifications to your `composer.json` file:
+
+*  Add a `repositories` section beneath the `config` section. If the `repositories` section already exists, add the lines to the bottom of the section.
+
+   ```json
+   "repositories": {
+    "module-adobe-io-events": {
+        "type": "git",
+        "url": "git@github.com:magento/module-adobe-io-events.git"
+    },
+    "event-plugin-generator": {
+        "type": "git",
+        "url": "git@github.com:magento/event-plugin-generator.git"
+    },
+    "module-commerce-events-client": {
+        "type": "git",
+        "url": "git@github.com:magento/module-commerce-events-client.git"
+    }
+   },
+   ```
+
+*  Add the following lines to the bottom of the `require` section.
+
+   ```json
+   "magento/module-adobe-io-events": "dev-master as 0.99",
+   "magento/module-commerce-events-client": "dev-main as 0.99",
+   "magento/event-plugin-generator": "dev-main as 0.99"
+   ```
+
+*  Find the `prefer-stable` line and modify the minimum stability:
+
+   ```json
+   "prefer-stable": true,
+   "minimum-stability": "dev"
+   ```
+
+Save your changes. The remaining installation steps vary, depending on your environment.
+
+### Cloud installation
+
+1. Update your `.magento.app.yaml` file to include the following build hook:
+
+```yaml
+hooks:
+  build: |
+   set -e
+   composer install
+   php ./bin/magento events:generate:module
+   php ./bin/magento module:enable Magento_AdobeCommerceEvents
+   php ./bin/magento module:enable Magento_AdobeCommerceEventsClient
+   php ./bin/magento module:enable Magento_AdobeCommerceEventsGenerator
+   php ./vendor/bin/ece-tools run scenario/build/generate.xml
+   php ./vendor/bin/ece-tools run scenario/build/transfer.xml
+```
+
+This hook generates and enables the `AdobeCommerceEvents` module, which allows you to register custom events.
+
+<!--Note to reviewer: I deleted references of bin/magento events:metadata:populate because I'm going to assume that a custom module with specific pre-defined events isn't available at initial installation/configuration. I will create a separate "Module development/integration" topic that addresses this command. -->
+
+### Local and on-premises installation
+
+1. Update the project dependencies.
+
+  ```bash
+  composer update
+  ```
+
+1. Run the following command to generate the `AdobeCommerceEvents` module. This module helps register  events. At this point, the module is empty.
+
+   ```bash
+   bin/magento events:generate:module
+   ```
+
+1. Enable the newly-installed modules:
+
+   ```bash
+   bin/magento module:enable --all
+   ```
+
+1. Upgrade your instance:
+
+   ```bash
+   bin/magento setup:upgrade
+   ```
+
+1. Compile your instance to generate new classes:
+
+   ```bash
+   bin/magento setup:di:compile
+   ```
+
+## Begin configuring events on Commerce
+
+You must configure Commerce to communicate with your project. You will need two files that you downloaded from the the Adobe Developer Console.
```

Comment:
```suggestion
You must configure Commerce to communicate with your project. You will the need two files that you downloaded from the the Adobe Developer Console.
```

Should we name the files here? or maybe point to the sections where downloading each file is discussed ([Set up a project](https://developer-stage.adobe.com/commerce/extensibility/events/project-setup/#set-up-a-project) (last step) and [Download the workspace configuration file](https://developer-stage.adobe.com/commerce/extensibility/events/project-setup/#download-the-workspace-configuration-file)